src/rgw/CMakeLists.txt

```diff
@@ -1,20 +1,20 @@
 if(WITH_TESTS)
-add_executable(ceph_rgw_jsonparser
-  rgw_jsonparser.cc)
-target_link_libraries(ceph_rgw_jsonparser
-  rgw_a
-  global)
+  add_executable(ceph_rgw_jsonparser
```

Comment:
please do not change formatting, it makes it harder to review actual code change.
you can do it in  a separate cleanup patch